src/hooks/useBookingForm.js

```diff
@@ -0,0 +1,46 @@
+import { useForm } from 'react-hook-form';
```

Comment:
If this component is only used in the `BookingCard`, it shouldn't be kept in the reusable `hooks` directory.
I suggest creating a `BookingCard` directory and keeping it there.